gulp/tasks/jquery.uls.js

```diff
@@ -0,0 +1,24 @@
+import gulp from 'gulp';
+import loadPlugins from 'gulp-load-plugins';
+import config from '../config.js';
+const plugins = loadPlugins();
+
+
+//--------------------------------------------------------
+// Concatenate jquery-uls libraries
+//--------------------------------------------------------
+const jqueryUlsPath = [
+  'node_modules/@bower_components/jquery/dist/jquery.js',
```

Comment:
Could we just as easily remove the `@bower_components/` prefix both here and in package.json, and this would still work?

Reply:
No, earlier I did the same thing and tried to use the same jquery packages here and that's what to lead the survey errors. I tried it with bower-away and seemed to work. If we use the same package here it leads to jquery declaration again and leads to those errors though I am not sure.

Reply:
I see. So maybe we can do that separately later on via getting rid of the webpack `externals` for jquery and explicitly requiring jquery everywhere it's needed.

Reply:
Yes, we can try that later though I haven't looked much into it. Will try to take a look and see what alternatives we have so that we don't need to manage separate modules for these bower modules.

Reply:
I think a good strategy will be to modify `views/shared/_head.html.haml` to only include the jquery-uls.js if `Features.enable_language_switcher?`, then fix all the ensuing chaos. ;)

Reply:
@ragesoss I will try getting rid of the jquery from externals and bundling it with webpack as well and then add the check for language switcher and try to look into the issues from surveys. I will update you about the status.